ansible/roles/prosody/templates/prosody.cfg.lua.j2

```diff
@@ -772,6 +772,8 @@ Component "visitors.{{ prosody_domain_name }}" "visitors_component"
     {% endif %}
     {% if prosody_visitors_always_supported %}
     always_visitors_enabled = true
+    {% else %}
```

Comment:
Any reason not to simplify to `always_visitors_enabled = {{ prosody_visitors_always_supported | to_json }}`

Reply:
Nope, updated it..